### PR TITLE
Reset state and logic to reset when out of sync

### PR DESCRIPTION
**Pull request by VeliML**
Mitsurunner will go to reset state if defrosting has started in unexpected state.

### CODE REVIEW

File: mitsurunner.yaml [hunk 1]
```diff
@@ -128,7 +128,7 @@ sensor:
                 // Wait first that sensors has read some values.
                 if (G_state_time_passed) {
                     // If defrosting is going on
-                    if (temperature_delta <= 0.0) {
+                    if (isisDefrostingOngoing) {
```
Comment (VeliML) [inline mitsurunner.yaml]: Corrected on newest file

File: mitsurunner.yaml [hunk 2]
```diff
@@ -118,7 +118,7 @@ sensor:
 # This C++ code checks if all requirements are fullfilled to trigger defrosting and triggers it then.
     lambda: !lambda |-
         float temperature_delta = (id(outdoor_temp).state - id(heat_exchanger_temp).state);
-
+        bool isisDefrostingOngoing = (temperature_delta <=0 )? true:false;
```
Comment (VeliML) [inline mitsurunner.yaml]: Corrected on newest file

**Comment by kestopuu** [comment]
Let's reject this. It's deprecated and not sure if needed. It's easy to add later.

**Comment by kestopuu** [comment]
Deprecated.

**Pull request closed without merging**